src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -168,29 +172,22 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
     if (this._hass) {
       element.hass = this._hass;
     }
+
     if (
-      entityConf.entity &&
-      !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityConf.entity))
+      !entityConf.tap_action ||
+      (entityConf.tap_action && entityConf.tap_action.action !== "none")
     ) {
       element.classList.add("state-card-dialog");
```

Comment:
Why do we only add this if there is a default action? It feels weird that this element should even know about any clicking being added, should be responsibility of generic-entity-row ?

Reply:
Not all have generic-entity-row, input-select-entity-row in particular does not use generic-entity-row

Reply:
Actually, I wonder if the longpress directive should add this?

Reply:
hm, no that feels weird, I guess the component that renders a longpress directive should take care of it